pkg/registry/apis/provisioning/repository/github.go

```diff
@@ -648,6 +648,7 @@ func (r *githubRepository) parsePullRequestEvent(event *github.PullRequestEvent)
 	return &provisioning.WebhookResponse{
 		Code:    http.StatusAccepted, // Nothing needed
 		Message: fmt.Sprintf("pull request: %s", action),
+		JobName: fmt.Sprintf("%s-%s-%d", r.Config().GetName(), provisioning.JobActionPullRequest, pr.GetNumber()),
```

Comment:
all the files you construct the `JobName` are already part of the `WebhookResponse`. Let's deal with the naming in `webhook.go` or the queue itself instead.